### PR TITLE
Feat: add API keys UI to Circle Admin page

### PR DESCRIPTION
**Pull request by META-DREAMER**
## Motivation and Context

Moves circle API key functionality from the hidden api page to the main circle admin UI + add form components for the new contributions API from PR #1609 

## Description

Removed Circle API key page and reimplemented UI for display in circle admin settings.

## Test and Deployment Plan

Create and delete API keys from the circle admin page and ensure there are no bugs / UX makes sense. 

## Screenshots (if appropriate):
CleanShot 2022-10-31 at 22 24 57

## Reviewers
@reeserj 

## Related Issue
#1212

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
     "react-transition-group": "^4.4.5",
     "react-virtualized-auto-sizer": "1.0.5",
     "recoil": "0.4.1",
-    "sharp": "^0.30.5",
+    "sharp": "^0.31.2",
```
Comment (META-DREAMER) [inline package.json:81]: had to be updated due to issue with ARM machines https://github.com/lovell/sharp/issues/2340

**Review by topocount — COMMENTED** [review]
This looks good to me but I think @wingfeatherwave and/or @ivyquinzel might want to weigh in

**Comment by wingfeatherwave** [comment]
@META-DREAMER I have not been able to successfully create an api key (pulled the branch locally).  Any setup I need to do to make it work?

**Comment by wingfeatherwave** [comment]
@META-DREAMER I also pushed up some fixes for tooltip positioning (will require yarn install) and other minor UI stuff.  

Still wasn't able to get an API key generated.

**Review by topocount — APPROVED** [review]
This works now for me. Able to generate an API key after adding the changes in https://github.com/coordinape/coordinape/pull/1607/commits/a6e066c4f18d91e572936dde257613b1a74c569d

**Pull request merged**